Representable/representable
# Chuk bug fixes

**changes**
- Fixes the dropdown aesthetic issue on Safari on the entry survey page

**to test**
- Go to submit an entry on safari and check that the dropdown and other portions of the entry page looks normal
